src/Custom-sample.h

```diff
@@ -1,320 +0,0 @@
-// #define USES_P063   // TTP229_KeyPad
-// #define USES_P064   // APDS9960 Gesture
-// #define USES_P065   // DRF0299
-// #define USES_P066   // VEML6040
-// #define USES_P067   // HX711_Load_Cell
-// #define USES_P068   // SHT3x
-// #define USES_P069   // LM75A
-
-// #define USES_P070   // NeoPixel_Clock
-// #define USES_P071   // Kamstrup401
-// #define USES_P072   // HDC1080
-// #define USES_P073   // 7DG
-// #define USES_P074   // TSL2561
-// #define USES_P075   // Nextion
-// #define USES_P076   // HWL8012   in POW r1
-// #define USES_P077   // CSE7766   in POW R2
-// #define USES_P078   // Eastron Modbus Energy meters
-// #define USES_P079   // Wemos Motoshield
-
-// #define USES_P080   // iButton Sensor  DS1990A
-// #define USES_P081   // Cron
-// #define USES_P082   // GPS
-// #define USES_P083   // SGP30
-// #define USES_P084   // VEML6070
-// #define USES_P085   // AcuDC24x
-// #define USES_P086   // Receiving values according Homie convention. Works together with C014 Homie controller
-// #define USES_P087   // Serial Proxy
-
-
-// Special plugins needing IR library
-// #define USES_P016   // IR
-// #define P016_SEND_IR_TO_CONTROLLER false //IF true then the JSON replay solution is transmited back to the condroller.
-// #define USES_P035   // IRTX
-// #define P016_P035_Extended_AC // The following define is needed for extended decoding of A/C Messages and or using standardised 
-                                 //common arguments for controlling all deeply supported A/C units
-// #define P016_P035_USE_RAW_RAW2 //Use the RAW and RAW2 encodings, disabling it saves 3.7Kb
-// #define USES_P088   // Heatpump IR
-
-
-/*
- #######################################################################################################
- ###########     Controllers
- #######################################################################################################
- */
-
-
-// #define USES_C001   // Domoticz HTTP
-// #define USES_C002   // Domoticz MQTT
-// #define USES_C003   // Nodo telnet
-// #define USES_C004   // ThingSpeak
-// #define USES_C005   // Home Assistant (openHAB) MQTT
-// #define USES_C006   // PiDome MQTT
-// #define USES_C007   // Emoncms
-// #define USES_C008   // Generic HTTP
-// #define USES_C009   // FHEM HTTP
-// #define USES_C010   // Generic UDP
-// #define USES_C011   // Generic HTTP Advanced
-// #define USES_C012   // Blynk HTTP
-// #define USES_C013   // ESPEasy P2P network
-// #define USES_C014   // homie 3 & 4dev MQTT
-// #define USES_C015   // Blynk
-// #define USES_C016   // Cache controller
-// #define USES_C017   // Zabbix
-// #define USES_C018   // TTN/RN2483
-
-
-/*
- #######################################################################################################
- ###########     Notifiers
- #######################################################################################################
- */
-
-
-// #define USES_N001   // Email
-// #define USES_N002   // Buzzer
-
-
-#endif // ESPEASY_CUSTOM_H
```

Comment:
You did delete this file, but it should be in the repository.

Reply:
Ohh i remaned it as in the description proposed. Now the original is again in the repository
thx